docs/ocis/development/beta-testplan.md

```diff
@@ -0,0 +1,313 @@
+
+| Test Case                                                                                                          | Expected Result                                                        | Comment |
+|--------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------|---------|
+| Login with the created user                                                                                        | Admin logs in.                                                         |         |
+| Create a text file.                                                                                                | Text editor can open, file is saved.                                   |         |
+| Create a text file with special characters as name                                                                 | file is created if the name is legal otherwise an error is displayed   |         |
+| Modify a text file.                                                                                                | File can be modified, no problems found.                               |         |
+| Rename a file.                                                                                                     | File is renamed.                                                       |         |
+| Upload a file.                                                                                                     | File is uploaded, no problems found.                                   |         |
+| Upload multiple files at once.                                                                                     | Files are uploaded, no problems found.                                 |         |
+| delete all content of a folder at once.                                                                            | Folder is cleanded, items are visible in the trashbin                  |         |
+| Overwrite a file by uploading a new version.                                                                       | File is uploaded and overwritten, file versions are displayed          |         |
+| Overwrite a file by uploading a new version, restore the original version.                                         | File is restored correctly                                             |         |
+| upload a huge file                                                                                                 | File is uploaded, no problems found.                                   |         |
+| upload a huge file, cancel the upload, restart the upload                                                          | File is uploaded, no problems founnd.                                  |         |
```

Comment:
upload should continue at the position where it was cancelled (TUS supports resumable uploads)